### PR TITLE
fix #3232 dropping only cobblestone

### LINKED ISSUE #3232: Andesite, Granite and Diorite drop cobblestone.

### PR DESCRIPTION
**Pull request by Yosshi999**

### CODE REVIEW

File: src/pocketmine/block/Stone.php [hunk 1]
```diff
@@ -77,12 +77,18 @@ public function getBreakTime(Item $item){
 
 	public function getDrops(Item $item){
 		if($item->isPickaxe() >= Tool::TIER_WOODEN){
-			return [
-				[Item::COBBLESTONE, 0, 1],
```
Comment (PEMapModder) [inline src/pocketmine/block/Stone.php]: Why not simply replace this line into `return [[Item::COBBLESTONE, $this->meta, 1]];`?

Comment (sekjun9878) [inline src/pocketmine/block/Stone.php]: Personally I wouldn't put two array brackets like that - it's not obvious that the array is supposed to be a list.

**Comment by shoghicp** [comment]
This is being changed in new-blocks

File: src/pocketmine/block/Stone.php (hunk 1 above)
Comment (Yosshi999) [inline src/pocketmine/block/Stone.php]: >>PEMapModder
Because the id of andesite etc is 1, not 4.(4 is cobblestone's id)

Comment (PEMapModder) [inline src/pocketmine/block/Stone.php]: @Yosshi999 from what I see in your code implementation, there is no difference.

**Pull request closed without merging**